.gas-snapshot

```diff
@@ -15,7 +15,7 @@ UInt256ArrayLib_sum:test_library() (gas: 16743)
 UInt256ArrayLib_sum:test_reference() (gas: 33815)
 BridgedPolygonNORI_name:test() (gas: 17211)
 BridgedPolygonNORI_permit:test() (gas: 91892)
-Certificate__msgSenderERC721A:test() (gas: 354)
+Certificate**msgSenderERC721A:test() (gas: 354)
```

Comment:
fyi i think something got messed up in this file-- can you regenerate it before merging?